src/irr/asset/CSmoothNormalGenerator.cpp

```diff
@@ -7,6 +7,12 @@ namespace irr
 namespace asset
 {
 
+static inline bool compareVertexPosition(const core::vectorSIMDf& a, const core::vectorSIMDf& b, float epsilon)
```

Comment:
this needs to be the default to the `calculateSmoothNormals` function's std::function parameter, need to support user define-able vertex comparison criterions